services/task-svc/internal/bed/bed.go

```diff
@@ -55,8 +54,6 @@ func (ServiceServer) CreateBed(ctx context.Context, req *pb.CreateBedRequest) (*
 		Str("name", bed.Name).
 		Msg("bed created")
 
-	_ = events.DispatchBedCreatedEvent(ctx, bed.ID, bed.Name)
```

Comment:
why?

Reply:
The initial usage of `hwes` in `task-svc` was just testing, not for event sourcing purposes.